Before: 7f01a519428dbf523f21d77ad05cef17b9606df319d5974d7710d7421bb52c49
After: b73de844401e0401e0c97dc556286c7192a90ac56bf4c3a0a7051f44386d8fdb
Patch: b89955f5bd930e75e81bef2b1fc3a4353fa47717f5c818c133a4583d2948c102
--- corvid-features/corvid.rb	2012-08-09 13:23:06.417199997 +1000
+++ corvid-features/corvid.rb	2012-08-09 13:23:43.907404464 +1000
@@ -1,11 +1,4 @@
 install {
   copy_file       'Rakefile'
   empty_directory 'lib'
-  empty_directory 'wtffff'
-}
-
-update {|ver|
-  if ver == 2
-    #empty_directory 'wtffff'
-  end
 }
